frontend/src/components/about/about_us.js

```diff
@@ -38,117 +61,130 @@ const AboutUs = () => {
           </div>
         </div>
       </div>
-      <div className="team container">
-        <h1>Meet The Team</h1>
-        <div className="dash"></div>
-        <h5 className="by-line">
-          “Individuals can and do make a difference, but it takes a team to
-          really mess things up.”
-        </h5>
+      <div className="meet">
         <div className="row">
-          <div className="col-lg-4">
-            <Card>
-              <ImageHeader imageSrc="https://cdn.mos.cms.futurecdn.net/3kZ3hc2YMB6LXiPohtyfKa.jpg" />
-              <CardBody>
-                <h4>Name</h4>
-                <p>
-                  Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nam
-                  vero.
-                </p>
-              </CardBody>
-              <CardFooter>
-                <i className="fab fa-linkedin fa-2x card_footer in"></i>
-                <i className="fab fa-twitter-square fa-2x card_footer"></i>
-                <i className="fab fa-github-square fa-2x card_footer"></i>
-              </CardFooter>
-            </Card>
+          <div className="col-lg-4 text-center">
+            <h1>MEET THE CREW</h1>
+            <h5 >
+              “Individuals can and do make a difference, but it takes a team to
+              really mess things up.”
+            </h5>
           </div>
           <div className="col-lg-4">
-            <Card>
-              <ImageHeader imageSrc="https://cdn.mos.cms.futurecdn.net/3kZ3hc2YMB6LXiPohtyfKa.jpg" />
-              <CardBody>
-                <h4>Name</h4>
-                <p>
-                  Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nam
-                  vero.
-                </p>
-              </CardBody>
-              <CardFooter>
-                <i className="fab fa-linkedin fa-2x card_footer in"></i>
-                <i className="fab fa-twitter-square fa-2x card_footer"></i>
-                <i className="fab fa-github-square fa-2x card_footer"></i>
-              </CardFooter>
-            </Card>
+            <img src="./images/profile.png" alt="" className="team-img"/>
```

Comment:
Instead of these static images, let's put the `Founder` and `Co-founder` details in here similar to other team members

Reply:
`Founder` and `Co-founder` team cards you mean just like the other cards below it?

Reply:
yes just their role would be `Founder` and `Co-founder`. Btw can be please have social handles visible only if someone hovers over the card?